spec/controllers/passwords_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+require 'devise'
+
+RSpec.describe PasswordsController, type: :controller do
+  describe 'POST create' do
+    let!(:user)  { create :user }
+    let!(:deactivated_user) { create :deactivated_user }
+    let(:params){
+      {
+        user:{
+          email: deactivated_user.email
+        }
+      }
+    }
+
+    context 'user is deactivated' do
+      it 'redirect and shows message' do
```

Comment:
Can we have something simple in here that differentiates it from the context below in terms of expectation? Otherwise it looks like we expect the same thing, so if this one also said "sends email" that'd help.